psychopy/visual/window.py

```diff
@@ -668,7 +668,7 @@ def flip(self, clearBuffer=True):
         if self.useFBO:
             if flipThisFrame:
                 self._prepareFBOrender()
-                # need blit the frambuffer object to the actual back buffer
+                # need blit the framebuffer object to the actual back buffer
```

Comment:
what is `blit` here?

Reply:
Blit is fine - it's a rendering term https://en.wikipedia.org/wiki/Bit_blit